core/lib/config/src/configs/api.rs

```diff
@@ -46,6 +46,107 @@ impl ApiConfig {
     }
 }
 
+/// Port binding specification.
+///
+/// Supports any of 3 formats:
+///
+/// - Just a `u16` port. This will bind a server to all IPv4 interfaces (i.e., `0.0.0.0`) for backward compatibility.
+/// - Full socket address (e.g., `127.0.0.1:8080`).
+/// - (For Unix systems) Path to a Unix domain socket (UDS) prefixed by `ipc:` (e.g., `ipc:chains/era/health.sock` or `ipc:/var/zksync.sock`).
+///   If the path is relative, it will resolve relative to the current working directory.
+#[derive(Debug, Clone, PartialEq)]
+pub enum BindAddress {
+    Tcp(SocketAddr),
+    #[cfg(unix)]
```

Comment:
Does this sort of conditional compilation make sense, or do we not care about non-Unix targets at this point?

Reply:
IMO let's keep it since it's pretty isolated and doesn't add almost any code